### PR TITLE
Added New Component: TabView

### PR DESCRIPTION
**Pull request by arpitBhalla**
**Summary**
https://material-ui.com/components/tabs/
https://material.io/components/tabs

- [x] Component
- [x] Tests
- [x] Docs

### CODE REVIEW

**Comment by flyingcircle** [comment]
How is this different from `ButtonGroup`?

**Comment by arpitBhalla** [comment]
FWojY
AFgz0

**Comment by arpitBhalla** [comment]
@pranshuchittora Should I continue on this?

**Comment by arpitBhalla** [comment]
@flyingcircle Should I continue on this?

**Comment by flyingcircle** [comment]
I think as an alternative styling of `ButtonGroup` this could be a nice simple addition. Sure.

**Comment by arpitBhalla** [comment]
@flyingcircle  Please Review the props and code

**Comment by arpitBhalla** [comment]
@flyingcircle Please Review

File: src/index.ts [hunk 1]
```diff
@@ -146,3 +146,5 @@ export type {
   SpeedDialProps,
   ThemeProps,
 };
+export { default as Tab } from './tab/tab';
```
Comment (flyingcircle) [inline src/index.ts:149]: Why not follow the pattern of the rest of the file? Please use the same pattern as every other component.

**Review by flyingcircle — CHANGES_REQUESTED** [review]

**Pull request merged**